src/index.ts

```diff
@@ -430,7 +430,7 @@ export class ERC725 {
   static encodePermissions(permissions: Permissions): string {
     const result = Object.keys(permissions).reduce((previous, key) => {
       return permissions[key]
-        ? previous + hexToNumber(LSP6_DEFAULT_PERMISSIONS[key])
+        ? previous + Number(hexToNumber(LSP6_DEFAULT_PERMISSIONS[key]))
```

Comment:
Why is this casting necessary? Doesn't the function `hexToNumber` convert to a `Number` already?

Reply:
The type has changed in v1.9.0 -> https://github.com/web3/web3.js/releases/tag/v1.9.0

> utils.toNumber and utils.hexToNumber can now return the large unsafe numbers as BigInt, if true was passed to a new optional parameter called bigIntOnOverflow (https://github.com/web3/web3.js/pull/5845)

We dont call it with true so it should not output a BigInt, but still the TS type thinks it is Number or string. This is to limit to Number only (which should be the truth)